util/flags.py

```diff
@@ -22,7 +22,7 @@ def create_flags():
     # Global Constants
     # ================
 
-    tf.app.flags.DEFINE_integer ('epoch',            75,          'target epoch to train - if negative, the absolute number of additional epochs will be trained')
+    tf.app.flags.DEFINE_integer ('epoch',            75,          'how many epochs (complete runs through the train files) to train for')
```

Comment:
I think we should rename it to `epochs` or `num_epochs` now.

Reply:
Done in 97c3629